.github/labeler.yml

```diff
@@ -0,0 +1,3 @@
+computeReview:  
+- 'specification/compute/*'
+- 'specification/compute/**/*'
```

Comment:
Why do you need both these file paths? Isn't one a subset of the other?

Reply:
all the examples I looked at seemed to do that to ensure all the files/folders within the path are labeled https://github.com/actions/labeler for example